core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1121,6 +1121,17 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_DAILY_BACKUP_HOUR =
+      new Builder(Name.MASTER_DAILY_BACKUP_HOUR)
+          .setDefaultValue("4")
+          .setDescription("Default hour for writing master metadata backups. "
+              + "This hour is based on 24-hour clock. E.g., at 10:04:15.250 PM the hour is 22."
```

Comment:
Can we give minute granularity, e.g. `4:30`? Also need to specify which time zone this is relative to, should probably use UTC.

Reply:
Now I support the minute granularity, `5` or `5:03` both works. The time is also in UTC time zone. Thanks!